src/main/resources/application-worker.yaml

```diff
@@ -6,6 +6,7 @@ SERVICE_INSTANCE_INGRESS_DEAD_LETTER_TOPIC: ${clowder.kafka.topics[?(@.requested
 
 rhsm-subscriptions:
   hourlyTallyDurationLimitDays: 90d
+  hourlyTallyEventBatchSize: ${HOURLY_TALLY_EVENT_BATCH_SIZE:16000}
```

Comment:
See previous comment about the documenting why 16k. Also note because providing a default here takes precedence over the field default, you could omit the field default I think.

Reply:
DONE